AI-MLbenchmarks/llm-rag/deploy.sh

```diff
@@ -0,0 +1,46 @@
+# Clone the repo
+git clone https://github.com/kusumachalasani/llm-rag-deployment.git
+
+cd llm-rag-deployment/bootstrap-rag
+
+# Pre-requisites
+# Create the Imagestream that supports the demo
+oc apply -f rhoai-rag-configuration/workbench-imagestream.yaml
+
+# Install ODF and ceph-rbd-storage class. <TODO>
+# This is required for llm PVC to support multiprocessing
+# Has issues creating the storage system in AWS. 
+# Workaround to use gp3 which is default Storage system
+
+# Deploy the postres 
+
```

Comment:
These throw an error as the namespace doesn't exist, might be we need to move it after namespace creation

Reply:
Also add a script for cleaning up

Reply:
Done.